lib/Logger.js

```diff
@@ -30,14 +30,19 @@ class Logger {
      *                                      purposes (except that the 'level'
      *                                      field is not accounted for)
      *
+     * @param {string} subModuleName - The name of the sub module using this
+     *                                 logger. It can be found later on in the
+     *                                 log entries.
+     *
      * @see [Bunyan's documentation]{@link
      * https://github.com/trentm/node-bunyan/blob/master/README.md#streams} for
      * a more detailed description of the streams array configuration.
      *
      * @returns {undefined}
      */
-    constructor(name, config) {
+    constructor(name, config, subModuleName) {
```

Comment:
I'd have put subModule name as the second argument for consistency,  but doing it this way keeps the backwards compatibility so :+1: